doc/topics/releases/3005.rst

```diff
@@ -5,3 +5,122 @@ Salt 3005 Release Notes - Codename Phosphorus
 =============================================
 
 Salt 3005 is currently under development.
+
+New Packages Available
+----------------------
+
+With the release of Salt 3005 we are pleased to announce the
```

Comment:
We have been using 'Tiamat'  capitalized for the last few years, I think we sure continue using it that way since it is an architecture over several platforms.